src/patternfly/components/Table/table-grid.scss

```diff
@@ -32,7 +32,7 @@
 }
 
 // - Table grid
-.#{$table}[class*="pf-m-grid"] {
+:where(:root, .#{$table}[class*="pf-m-grid"]) {
```

Comment:
Does this one need to be closed after the variables then?

Reply:
It closes on [line 127](https://github.com/patternfly/patternfly/pull/6752/files#diff-1c73508a6dac9de0a0052bcd77f9193e0f8d95a4d5cfef221569b6b97af1e649L127). Is that what you're looking for?

Reply:
Ah, yeah sorry, the collapse gets me every time.